pkg-r/README.md

```diff
@@ -5,9 +5,11 @@
 [![CRAN status](https://www.r-pkg.org/badges/version/shinychat)](https://CRAN.R-project.org/package=shinychat)
 <!-- badges: end -->
 
-Chat UI component for [Shiny for R](https://shiny.posit.co/).
+**shinychat** is a chat UI component for [Shiny for R](https://shiny.posit.co/).
 
-(For [Shiny for Python](https://shiny.posit.co/py/), see [ui.Chat](https://shiny.posit.co/py/components/display-messages/chat/).)
+It’s also available for [Shiny for Python](https://posit-dev.github.io/shinychat/py/index.html).
+
+**Build your first shinychat chatbot:** See [Get Started](articles/get-started.html) to learn more.
```

Comment:
I don't feel strongly about this, but I was expecting to see it at the bottom of the page (in something like a next steps section). It also works here though